griptape/events/__init__.py

```diff
@@ -20,5 +21,7 @@
     "FinishPromptEvent",
     "StartStructureRunEvent",
     "FinishStructureRunEvent",
-    "CompletionChunkEvent"
+    "CompletionChunkEvent",
+
```

Comment:
extra space?

Reply:
I intentionally added it for organization since Event Listener is a different thing than the Events. We do it in [other places](https://github.com/griptape-ai/griptape/blob/main/griptape/drivers/__init__.py) too.

Though once we merge in the autoformatter these intentional whitespaces are going to get annihilated lol.